src/tools/compiletest/src/runtest.rs

```diff
@@ -1310,7 +1310,14 @@ impl<'test> TestCx<'test> {
                 }
 
                 None => {
-                    if self.is_unexpected_compiler_message(actual_error, expect_help, expect_note) {
+                    // If the test is a known bug, don't require that the error is annotated
```

Comment:
I might suggest we assert! that *no* expected errors are in the file -- otherwise, it might be easy to forget to drop known-bug and if nothing near it is edited GitHub won't really show that.